src/Microsoft.Identity.Web/WebApiAuthenticationBuilderExtensions.cs

```diff
@@ -28,7 +28,6 @@ public static partial class WebApiAuthenticationBuilderExtensions
         /// <param name="configuration">The Configuration object.</param>
         /// <param name="configSectionName">The configuration section with the necessary settings to initialize authentication options.</param>
         /// <param name="jwtBearerScheme">The JwtBearer scheme name to be used. By default it uses "Bearer".</param>
-        /// <param name="tokenDecryptionCertificate">Token decryption certificate (null by default).</param>
```

Comment:
The AddMicrosoftWebApi don't have the tokenDecryptionCertificate any longer. The AddProtectedWebApi still do.